Assets/UX/Scripts/LocalizationManager.cs

```diff
@@ -0,0 +1,260 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Localization;
+using UnityEngine.Localization.Settings;
+using UnityEngine.Localization.Tables;
+using UnityEngine.ResourceManagement.AsyncOperations;
+
+public class LocalizationManager : MonoBehaviour
+{
+    public enum SupportedLanguages
+    {
+        ChineseSimplified,

```

Comment:
Not too important, but in future localizations maybe using a standard for languages would make sense https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes